lib/src/device_info_model.dart

```diff
@@ -16,11 +16,14 @@ class DeviceInfoModel {
   final String? packageName;
   final String? version;
   final String? buildNumber;
-
+  
   final String? id;
   final int? androidSdkInt;
   final String? identifier;
 
+  final String? deviceName;
```

Comment:
please show case in example project for 2 information and update new picture screenshot
thanks